.github/workflows/build-windows.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
       - name: Restore Conda env cache
         id: conda-cache
-        uses: actions/cache@v3
+        uses: actions/cache/restore@v3
```

Comment:
same here, better to have the same action name in all places; if you add "restore" here, add everywhere

Reply:
done